modules/cxf/src/main/java/org/mule/module/cxf/CxfOutboundMessageProcessor.java

```diff
@@ -52,7 +55,7 @@
  * The CxfOutboundMessageProcessor performs outbound CXF processing, sending an event
  * through the CXF client, then on to the next MessageProcessor.
  */
-public class CxfOutboundMessageProcessor extends AbstractInterceptingMessageProcessor implements CloneableMessageProcessor
+public class CxfOutboundMessageProcessor extends AbstractInterceptingMessageProcessor implements CloneableMessageProcessor, NonBlockingSupported
```

Comment:
Inbound implements NonBlockingMP, but outbound implements NonBlockingSupported, this is somewhat inconsistent.